### DISCUSSION TITLE
TON blockchain support

### DISCUSSION THREAD

**Discussion by Chaps00** [Web3Auth/Web3Auth#1066] [Help]
Hello, Help me plz understand, is TON blockchain supported (via some special settings maybe)? or any plans to add it in future?

**Comment by Saviour1001** [comment]
Web3Auth is chain agnostic, we provide the infrastructure which allows any chain to integrate with it. 
Here is a [multichain](https://github.com/Web3Auth/examples/tree/main/web-modal-sdk/multi-chain/react-multi-chain-web3auth-example) example which showcases how we can create multiple chains wallets at once. 

To create a TON wallet using Web3Auth, all you need to do is integrate the TON SDKs which allow the creation of wallets, have a look at the multichain example, where we are using libraries from Tezos to create Tezos Wallet. 
I hope this helps!
